### PR TITLE
[faq] 🇮🇹 Added italian FAQ pages

### PR DESCRIPTION
**Pull request by lucaceccarini**
Added FAQ pages in italian language.

Signed-off-by: Luca Ceccarini <luca@decode.lc>

### CODE REVIEW

File: content/faq/app/how-to-use-android-auto/index.it.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: Come si usa Android Auto?
+description: "Domande frequenti sull'applicazione Organic Maps".
```
Comment (gpesquero) [inline content/faq/app/how-to-use-android-auto/index.it.md:3]:
```suggestion
description: "Domande frequenti sull'applicazione Organic Maps"
```

File: content/faq/app/which-ios-and-android-versions-are-supported/index.it.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+title: Quali versioni di iOS e Android sono supportate?
+description: "Domande frequenti sull'applicazione Organic Maps".
```
Comment (gpesquero) [inline content/faq/app/which-ios-and-android-versions-are-supported/index.it.md:3]:
```suggestion
description: "Domande frequenti sull'applicazione Organic Maps"
```

**Comment by gpesquero** [comment]
@lucaceccarini: You must also add in the `config.toml` file the following keys in the `[languages.it.translations]` section:
```
engines = "Motori TTS supportati"
faq-menu-title = "Domande frequenti"
language-word = "Lingua"
```
(Please confirm the translation to Italian)

File: content/faq/app/how-to-use-android-auto/index.it.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
```
Comment (gpesquero) [inline content/faq/app/how-to-use-android-auto/index.it.md:4]:
```suggestion
slug: come-si-usa-android-auto
```

File: content/faq/app/which-ios-and-android-versions-are-supported/index.it.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
```
Comment (gpesquero) [inline content/faq/app/which-ios-and-android-versions-are-supported/index.it.md:4]:
```suggestion
slug: quali-versioni-di-ios-e-android-sono-supportate
```

File: content/faq/text-to-speech-android-tts/index.it.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
+---
+title: Text-to-Speech su Android
+description: Guida su come far funzionare il TTS su Android
+
```
Comment (gpesquero) [inline content/faq/text-to-speech-android-tts/index.it.md:4]:
```suggestion
slug: text-to-speech-su-Android
```

**Review by gpesquero — CHANGES_REQUESTED** [review]
@lucaceccarini: Please add localized slug for better SEO

File: content/faq/app/how-to-use-android-auto/index.it.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+taxonomies:
+  faq: ["App"]
+
+extra:
+  order: 20
+---
+
+Per utilizzare Organic Maps con Android Auto, è necessario disporre almeno della versione 8.0 (Oreo) o successiva di Android.
+
+Per ulteriori informazioni, consultare il [sito web di Android Auto] (https://www.android.com/auto/).
```
Comment (gpesquero) [inline content/faq/app/how-to-use-android-auto/index.it.md:14]: @lucaceccarini: Please remove extra space:
```suggestion
Per ulteriori informazioni, consultare il [sito web di Android Auto](https://www.android.com/auto/).
```

File: content/faq/app/which-ios-and-android-versions-are-supported/index.it.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+extra:
+  order: 10
+---
+
+Per eseguire l'app è necessario avere almeno iOS 12 o Android 5. I dispositivi con versioni successive possono eseguire Organic Maps.
+
+Su Android, Organic Maps può funzionare sia su dispositivi con il supporto di Google Services, sia sui dispositivi senza il supporto di Google.
+
+Per utilizzare Organic Maps con Android Auto, verificare i requisiti [qui](../come-usare-android-auto/).
```
Comment (gpesquero) [inline content/faq/app/which-ios-and-android-versions-are-supported/index.it.md:16]: @lucaceccarini: This link shall point to the same `slug` stated in the page.
```suggestion
Per utilizzare Organic Maps con Android Auto, verificare i requisiti [qui](../come-si-usa-android-auto/).
```

File: content/faq/app/how-to-use-android-auto/index.it.md (hunk 3 above)
Comment (lucaceccarini) [reply content/faq/app/how-to-use-android-auto/index.it.md:5]: Yup, wasn't sure about the slug. Thanks!

**Comment by lucaceccarini** [comment]
> @lucaceccarini: You must also add in the `config.toml` file the following keys in the `[languages.it.translations]` section:

Done. Completely missed that, thanks!

File: content/faq/_index.it.md [hunk 8]
```diff
@@ -0,0 +1,9 @@
+---
+title: Domande frequenti
+description: Questa sezione contiene le risposte a molte domande sull'applicazione Organic Maps, sui nostri collaboratori e sul nostro progetto
```
Comment (biodranik) [inline content/faq/_index.it.md:3]: There is a single quote here that fails the yaml check: https://stackoverflow.com/a/22235064
```suggestion
description: "Questa sezione contiene le risposte a molte domande sull'applicazione Organic Maps, sui nostri collaboratori e sul nostro progetto"
```

File: content/faq/app/which-ios-and-android-versions-are-supported/index.it.md [hunk 9]
```diff
@@ -0,0 +1,17 @@
+---
+title: Quali versioni di iOS e Android sono supportate?
```
Comment (biodranik) [inline content/faq/app/which-ios-and-android-versions-are-supported/index.it.md:2]: People recommend using quotes for special characters: https://stackoverflow.com/a/22235064

**Comment by gpesquero** [comment]
Hi again @lucaceccarini: We have forgotten to translate also the names of the taxonomies, so the "App" and "Voice instructions" sections in the FAQ list and in the FAQ top menu are still displayed in English:

italian1

italian2

These translations shall be done in the "faq" field included in the header of each of the new .md files:
```
---
title: Text-to-Speech su Android
description: Guida su come far funzionare il TTS su Android
slug: text-to-speech-su-android

taxonomies:
  faq: ["Voice Directions"]      << Translate this !!

extra:
  order: 10
---
```

Please launch an additional PR with this translations, or you can provide the Italian translation of "App" and "Voice instructions" and I can launch the PR myself.

**Pull request merged**